Editor/UI/Wizard/Setup/XRSDKSetupPage.cs

```diff
@@ -67,6 +67,11 @@ public override void Draw(Rect window)
                         otherHardwareText = CreatorGUILayout.DrawTextField(otherHardwareText, -1,GUILayout.Width(window.width * 0.4f));
                     GUILayout.EndHorizontal();
                 }
+
+                if (selectedLoader == XRLoader.Oculus || selectedLoader == XRLoader.WindowsMR)
+                {
+                    EditorGUILayout.HelpBox("This selection will install Unity XR Toolkit, which includes Unity's new InputSystem. By installing this, the event handling will be switched, which disables UIs created with IMGUI.\n\nTo fix this go to 'ProjectSettings/Player/Other Settings' and set 'Active Input Handling' to 'Both'", MessageType.Warning);
+                }
```

Comment:
Actually, at the moment we show this warning, XR Interaction Toolkit is already installed, the message has no sense and can be hidden for users that either already has an XR configuration or select something else but Oculus and WindowsMR 😱